vllm/entrypoints/openai/api_server.py

```diff
@@ -154,25 +200,31 @@ async def authentication(request: Request, call_next):
         elif inspect.iscoroutinefunction(imported):
             app.middleware("http")(imported)
         else:
-            raise ValueError(f"Invalid middleware {middleware}. "
-                             f"Must be a function or a class.")
+            raise ValueError(
+                f"Invalid middleware {middleware}. Must be a function or a class."
+            )
 
     logger.info("vLLM API server version %s", vllm.__version__)
     logger.info("args: %s", args)
 
+    if args.privileged:
+        logger.warning(
+            "\n"
+            "##########################################################################\n"
+            "privileged mode enabled. This should only be used for development purpose.\n"
```

Comment:
If it's for development or debugging purposes, it should probably be called development or debugging mode, not privileged mode.

Reply:
Modified to debugging mode.